Week-1/Intermediate/Tables/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>HTML Tables</title>
+    <!-- styles -->
+    <style>
+      #table {
+        border: 2px solid black;
```

Comment:
try moving the css part to another file and importing it in html file , it will be helpful in case hmtl file is large
https://www.w3schools.com/css/css_howto.asp

Reply:
good job